pallets/creditcoin/src/lib.rs

```diff
@@ -297,30 +291,22 @@ pub mod pallet {
 		/// A deal order has been funded by a lender. This indicates that the lender
 		/// has initiated the actual loan by transferring the loan amount to the borrower
 		/// on an external chain.
-		/// [funded_deal_order_id, funded_deal_order]
-		DealOrderFunded(
-			DealOrderId<T::BlockNumber, T::Hash>,
-			DealOrder<T::AccountId, T::BlockNumber, T::Hash, T::Moment>,
-		),
+		/// [funded_deal_order_id]
+		DealOrderFunded(DealOrderId<T::BlockNumber, T::Hash>),
 
 		/// A deal order has been locked by a borrower. This indicates that the borrower
 		/// is preparing to make a repayment and locks the loan from being sold or transferred
 		/// to another party.
-		DealOrderLocked(
-			DealOrderId<T::BlockNumber, T::Hash>,
-			DealOrder<T::AccountId, T::BlockNumber, T::Hash, T::Moment>,
-		),
+		/// [deal_order_id]
+		DealOrderLocked(DealOrderId<T::BlockNumber, T::Hash>),
 
 		/// A deal order has been closed by a borrower. This indicates that the borrower
 		/// has repaid the loan in full and is now closing out the loan.
-		/// [closed_deal_order_id, closed_deal_order]
-		DealOrderClosed(
-			DealOrderId<T::BlockNumber, T::Hash>,
-			DealOrder<T::AccountId, T::BlockNumber, T::Hash, T::Moment>,
-		),
+		/// [closed_deal_order_id]
+		DealOrderClosed(DealOrderId<T::BlockNumber, T::Hash>),
 
 		/// A loan exemption has been granted by a lender. This indicates that the lender
-		/// is releasing some or all of the outstanding debt on the loan. The borrower
+		/// is releasing (some!!) or all of the outstanding debt on the loan. The borrower
```

Comment:
what is this addition for?

Reply:
That was a personal note that I forgot to take out. On the other hand, I'm probably wrong but, the original intention of exempt() was to allow the borrower to repay partially and the lender consenting; as the above comment suggests. This is not what happens so far. The docs are misleading or the impl incomplete.

Reply:
There is no check as to what the borrower repaid, it really just releases them from the debt. This will need to be re-worked in the future.